openshift-ci/common.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+ingress_node_firewall_dir="$(dirname $(readlink -f $0))"
+source ${ingress_node_firewall_dir}/../../common.sh
+source ${ingress_node_firewall_dir}/../../network.sh
```

Comment:
those scripts on the release repo right ?

Reply:
in dev-scripts repo
https://github.com/openshift-metal3/dev-scripts/blob/master/common.sh
https://github.com/openshift-metal3/dev-scripts/blob/master/network.sh